portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACUtils.macro

```diff
@@ -648,30 +642,41 @@ definition {
 
 		return localStorageItem;
 		''';
-
 		var localStorageItem = selenium.getEval("${javaScript}");
-
 		var localStorageItem = StringUtil.replace("${localStorageItem}", "\"", "");
 
 		return "${localStorageItem}";
 	}
 
 	macro getPageRefRequestPropertyValue {
 		var postDataText = ProxyUtil.getHarRecording("$..[?(@['pageref'] == '${pageRef}')].request[?(@.method == 'POST' && @.url contains '/${pageRef}')]..text");
-
 		var value = JSONUtil.getWithJSONPath("[${postDataText}]", "$..${property}");
 
 		return "${value}";
 	}
 
 	macro getRequestCommonPropertyValue {
 		var postDataText = ProxyUtil.getHarRecording("$..request..postData..text");
-
 		var value = JSONUtil.getWithJSONPath("[${postDataText}]", "$.[-1:]..${property}");
 
 		return "${value}";
 	}
 
+	macro harRecordingPause {
+		if (isSet(time)) {
+
+			// Pauseing for harRecording test for reuse purpose
+
+			Pause(locator1 = "${time}");
+		}
+		else if (!(isSet(time))) {
+
+			// Pausing for harRecording tests
+
+			Pause(locator1 = "5000");
+		}
+	}
+
```

Comment:
Hey @ViniLopes20 I think Calvin should open a ticket for these pauses, right?
Like this comment: 
`// It needs a 30 second pause which is the time required for the command to be executed. For more information about pause see LRAC-9518`